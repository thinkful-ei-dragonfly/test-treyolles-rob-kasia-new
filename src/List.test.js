import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './Card'
import List from './List'


describe('<List />', () => {

    // smoke test
    it('renders without crashing', () => {
        const div = document.createElement('div');
  
        // Pass required props in the usual way
            ReactDOM.render(<List header='hi' cards={[{id:'a', title:'hfgj', content:'fghjkhkj'},{id:'b', title:'hfgj', content:'fghjkhkj'}]} />, div);
  
        ReactDOM.unmountComponentAtNode(div);
    });
   
  
      it('renders without crashing snapshot', () => {
          // Render the component, as JSON
          const tree = renderer.create(<Card title="Amazing title" content="What!" />).toJSON();  // title="Amazing title" content="What!"
          // Check whether it matches the previous snapshot
          // Stored in __snapshots__/App.test.js.snap
          expect(tree).toMatchSnapshot(); 
      });
  
      
  });