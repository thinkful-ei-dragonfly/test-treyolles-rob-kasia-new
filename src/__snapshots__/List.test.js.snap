// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<List /> renders without crashing snapshot 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    Amazing title
  </h3>
  <p>
    What!
  </p>
</div>
`;
